text_handler.py

```diff
@@ -4,11 +4,28 @@
 from aiogram import types
 from aiogram.types import ParseMode
 from pytube import YouTube
+from urllib.request import urlopen
+from PIL import Image
 import config
 
 logger = logging.getLogger(__name__)
 
 
+def mkthumb(thumbnail_url, thumbnail_file):
+    # https://stackoverflow.com/a/52177551
+    img = Image.open(urlopen(thumbnail_url))
+    width, height = img.size
+    length = min(width,height)
+    # Crop to square
```

Comment:
it's generally a bad programming practice to leave comments in the code (you can read the book [Clean Code](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882) to learn more about it).
The way to avoid it is using **private methods**. You should use them instead of comments. Here's an example:

```
def mkthumb(thumbnail_url, thumbnail_file):
    img = Image.open(urlopen(thumbnail_url))
    width, height = img.size
    length = min(width, height)
    img = img.__crop_to_square((width, length))
    img.__resize_to(320, 320)
    img.save(thumbnail_file)
```
so `__crop_to_square` and `__resize_to`will be private methods located right under `mkthumb` method. The url to stackoverflow is **useful**, but i would put it on top of the method like so
```
# https://stackoverflow.com/a/52177551
def mkthumb(thumbnail_url, thumbnail_file):
```
i think it looks nicer this way

Reply:
Should be fine now. I kept `squarethumb.thumbnail((320,320), Image.ANTIALIAS)` because it's just only line.